doc/index.rst

```diff
@@ -233,6 +233,41 @@ On CI platform usually some variables are available for the convenience. Since a
 Still, you may want to pass some other vars. `DOCKER_RUN_OPTS='-e MY_VARIABLE_VALUE'` should do the trick.
 You can even set it to a specific value: `DOCKER_RUN_OPTS='-e MY_VARIABLE_VALUE=42'` (format varies per CI platform. These are Gitlab CI example).
 
+(Gitlab CI) Access to private repositories
+------------------------------------------
+
+Every one who opens PRs/MRs (merge requests, that's what Gitlab calls pull requests) to repositories where CI jobs access private Gitlab repos are now required to add ssh private keys on their repos, including the forks (the following steps are cumbersome workaround, hopefully only temporary).
```

Comment:
Instructions should be addressed to any repo where CI jobs need to access private upstream packages. What about:

_Repositories where CI jobs access private Gitlab repos are required to add ssh private keys on their CI settings. Matching public keys must be set as deploy keys on the private repos that need to be accessed._

Furthermore, the specific setup for a fork of some other repo should either use the same private key or be able to add the new public key to the upstream repos, shouldn't it? Not sure how/where to add this, maybe some note for setting up CI for forks at the end of the section?

Reply:
> Furthermore, the specific setup for a fork of some other repo should either use the same private key or be able to add the new public key to the upstream repos, shouldn't it? Not sure how/where to add this, maybe some note for setting up CI for forks at the end of the section?

That sounds even more advanced. Not sure if we need information for that in `industrial_ci`'s doc. In this PR, I've already gone far to walk through steps of how to add private and pub keys, which is not this package-specific and may be too much. You think that should be included here?

Reply:
Not really. I just mention it because you were mentioning PRs/MRs and forks in the original paragraph.